### PR TITLE
SAAS-735 percona portal logo

### PR DESCRIPTION
**Pull request by TomasBlazek3PG**
Add "Percona Portal" next to logo.

image

### CODE REVIEW

File: src/components/MenuBar/MenuBar.styles.ts [hunk 1]
```diff
@@ -5,10 +5,19 @@ export const getStyles = (theme: GrafanaTheme) => {
   const { border, colors } = theme;
 
   const link = css`
+    padding: 0 15px;
```
Comment (tiagomotasantos) [inline src/components/MenuBar/MenuBar.styles.ts:8]: Can a spacing variable be used here?

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]
Well done, good work 🚀  just a minor request

**Review by tiagomotasantos — APPROVED** [review]
🚀

File: src/components/MenuBar/MenuBar.styles.ts [hunk 2]
```diff
@@ -2,13 +2,22 @@ import { css } from 'emotion';
 import { GrafanaTheme } from '@grafana/data';
 
 export const getStyles = (theme: GrafanaTheme) => {
-  const { border, colors } = theme;
+  const { border, colors, spacing } = theme;
 
   const link = css`
+    padding: 0 ${spacing.md};
+    text-decoration: none;
+    color: ${ colors.text };
```
Comment (nicolalamacchia) [inline src/components/MenuBar/MenuBar.styles.ts:10]: nit
```suggestion
    color: ${colors.text};
```

**Review by nicolalamacchia — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**